docs/extended_api/synchronization_primitives.md

```diff
@@ -1,48 +1,33 @@
----
-parent: Extended API
-has_children: true
-has_toc: false
-nav_order: 3
----
+## Synchronization Primitives
 
-# Synchronization Primitives
+### Atomics
 
-## Atomics
+| [`cuda::atomic`]             | System-wide [`cuda::std::atomic`] objects and operations. `(class template)`                                   <br/><br/> 1.0.0 / CUDA 10.2 |
 
-| [`cuda::atomic`]             | System-wide [`cuda::std::atomic`] objects. `(class template)` |
+### Latches
 
-## Barriers
+| [`cuda::latch`]              | System-wide [`cuda::std::latch`] single-phase asynchronous thread coordination mechanism. `(class template)`   <br/><br/> 1.1.0 / CUDA 11.0 |
 
-| [`cuda::barrier`]            | System-wide [`cuda::std::barrier`] objects. `(class template)` |
+### Barriers
 
-## Latches
+| [`cuda::barrier`]            | System-wide [`cuda::std::barrier`] multi-phase asynchronous thread coordination mechanism. `(class template)`  <br/><br/> 1.1.0 / CUDA 11.0 |
 
-| [`cuda::latch`]              | System-wide [`cuda::std::latch`] objects. `(class template)` |
+### Semaphores
 
-## Semaphores
+| [`cuda::counting_semaphore`] | System-wide [`cuda::std::counting_semaphore`] primitive for constraining concurrent access. `(class template)` <br/><br/> 1.1.0 / CUDA 11.0 |
+| [`cuda::binary_semaphore`]   | System-wide [`cuda::std::binary_semaphore`] primitive for mutual exclusion. `(class template)`                 <br/><br/> 1.1.0 / CUDA 11.0 |
```

Comment:
"System-wide cuda::std::X"? This is the extended API section, what is this trying to say?

Reply:
It's an extended version of the thing that is scoped and thus can work across your entire system.